src/main/java/org/iac2/service/checking/plugin/implementation/subgraphmatching/RuleChecker.java

```diff
@@ -1,17 +1,18 @@
 package org.iac2.service.checking.plugin.implementation.subgraphmatching;
 
+import java.util.Iterator;
+
 import io.github.edmm.model.component.RootComponent;
 import io.github.edmm.model.relation.RootRelation;
 import org.iac2.common.model.InstanceModel;
 import org.iac2.common.model.compliancerule.ComplianceRule;
+import org.iac2.common.utility.EdmmGraphCreator;
```

Comment:
This import is new but doesn't seem to be used in the class ? Atleast from the github perspective

Reply:
I have no idea how this happened. It is used in the constructor. So not sure why that usage is not new, or how this import is new :\